float32[] joint_angles
---
geometry_msgs/Polygon joint_positions